lib/sycamore/sycamore/transforms/explode.py

```diff
@@ -36,6 +37,8 @@ def explode(parent: Document) -> list[Document]:
             cur = Document(element.data)
             cur.doc_id = str(uuid.uuid4())
             cur.parent_id = parent.doc_id
+            if isinstance(element, TableElement):
```

Comment:
why do we need special handling here?

Reply:
In a previous PR I changed the behavior of TableElement.text_representation to 
```
if overridden: overridden_text
else if table: table.to_csv()
else: text_representation
```
(and setting text_representation sets the overridden).

But just importing the underlying data essentially deletes this behavior when the table gets promoted to a document